src/app/wizard/services/app-launcher-dependency-check.service.ts

```diff
@@ -15,8 +15,8 @@ export class AppLauncherDependencyCheckService implements DependencyCheckService
    */
   public getDependencyCheck(): Observable<DependencyCheck> {
     return of({
-      mavenArtifact: 'booster',
-      groupId: 'io.openshift.booster',
+      mavenArtifact: undefined,
```

Comment:
`undefined`?

Reply:
yes, the default is it's then set when the "booster" is selected to `newapp-${artifactMission}-${artifactRuntime}`

Reply:
see https://github.com/fabric8-launcher/ngx-launcher/pull/530/files